contentcuration/contentcuration/views.py

```diff
@@ -48,50 +50,53 @@ def testpage(request):
 @login_required
 def channel_list(request):
     channel_list = Channel.objects.filter(deleted=False, editors__email__contains= request.user)
-    channel_serializer = ChannelSerializer(channel_list, many=True)
+    channel_list = ChannelListSerializer.setup_eager_loading(channel_list)
+    channel_serializer = ChannelListSerializer(channel_list, many=True)
 
-    licenses = License.objects.all()
-    license_serializer = LicenseSerializer(licenses, many=True)
+    licenses = get_or_set_cached_constants(License, LicenseSerializer)
     return render(request, 'channel_list.html', {"channels" : JSONRenderer().render(channel_serializer.data),
-                                                 "license_list" : JSONRenderer().render(license_serializer.data),
+                                                 "license_list" : licenses,
                                                  "current_user" : JSONRenderer().render(UserSerializer(request.user).data)})
+
 @login_required
 def channel(request, channel_id):
     channel = get_object_or_404(Channel, id=channel_id, deleted=False)
     channel_serializer =  ChannelSerializer(channel)
 
-    channel_list = Channel.objects.filter(deleted=False, editors__email__contains= request.user)
-    channel_list_serializer = ChannelSerializer(channel_list, many=True)
-
     accessible_channel_list = Channel.objects.filter( Q(deleted=False, public=True) | Q(deleted=False, editors__email__contains= request.user))
-    accessible_channel_list_serializer = ChannelSerializer(accessible_channel_list, many=True)
-
-    fileformats = FileFormat.objects.all()
-    fileformat_serializer = FileFormatSerializer(fileformats, many=True)
+    channel_list = accessible_channel_list.filter(editors__email__contains= request.user)
```

Comment:
This can likely just be `editors__email=request.user`, or maybe just `editors=request.user`. It does a magic "existence check" for relation lookups like that.

(and if you do `email__contains` like that, it might match for user `ja@q.com` if there's an editor named `qqja@q.com`)